addons/test_mass_mailing/tests/test_performance.py

```diff
@@ -51,7 +51,7 @@ def test_send_mailing(self):
         })
 
         # runbot needs +50 compared to local
-        with self.assertQueryCount(__system__=1665, marketing=1665):
+        with self.assertQueryCount(__system__=1715, marketing=1715):
```

Comment:
how is a removal adding queries? is the new group related to mass mailing in any way?

Reply:
Those querie are there because of the field "mail_message_id" on the "mail.group.message" model... When we unlink the record, 1 SQL query is made to know if a "mail.group.message" is related to this "mail.message" :/

Reply:
Arf. So if mailing list is not installed count should not increase ? We could probably have a note for that .